package/ios/RNSkia-iOS/SkiaCVPixelBufferUtils.mm

```diff
@@ -124,7 +124,7 @@
 // pragma MARK: getTextureCache()
 
 CVMetalTextureCacheRef SkiaCVPixelBufferUtils::getTextureCache() {
-  static thread_local CVMetalTextureCacheRef textureCache = nil;
```

Comment:
The `thread_local` here was also unnecessary. This could really explode memory if used from different threads, so we want to keep a global instance of that as it is thread-safe 👍

Reply:
For the same reason as mentioned in https://github.com/Shopify/react-native-skia/pull/2368 this would cause the `CVMetalTextureCache` to be created multiple times (up to 10 times) in VisionCamera.

After this change, I shaved off 200 MB RAM (9 * 10 * 2,5MB) usage in VisionCamera.